### DISCUSSION TITLE
ModuleNotFoundError: No module named 'requests'

### DISCUSSION THREAD

**Discussion by Stevenca-t** [nianeyna/ao3downloader#73] [Q&A] [answered]
Whenever I try to open the script, this error message pops up and I'm unsure on how to fix it. Any help would be greatly appricated.

**Comment by nianeyna** [comment]
Make sure you check which version of python you have - the latest version does not currently work with the script. There's a link to a working version in the readme.

**Reply by soph104** [reply]
I've tried three different versions. I've also tried installing requests. Same error

**Reply by soph104** [reply, marked as answer]
Ah nevermind, finally got it to work in version 3.9.0

**Reply by seadeepy** [reply]
Thanks,, I had the same problem and this fixed it!

**Reply by alexandra-e-ressler** [reply]
Update:  I think it's working now!  After doing all the below, I redownloaded from github and clicking command seems to have installed itself.  I didn't need to do python3 -m venv venv etc. 

I got this requests message with https://www.python.org/downloads/release/python-3114

I tried installing 3.9.0 but now when I run the cmd it says 

No Python at '"C:\Users\alexa\AppData\Local\Programs\Python\Python311\python.exe'

I reinstalled 3114 and made sure to check "Add Python to environment variables"

Still getting ModuleNotFoundError: No module named 'requests'

For other reference, I've already done the  "Manage app execution aliases". There should be two aliases for Python. Unselect them, and this will allow the usual Python aliases "python" and "python3". thing.

I'm on Windows 11.

Any ideas for next steps?  Thank you!

**Reply by ComealongPond1** [reply]
I'm in the same boat still. Tried to downgrade to 3.9.0 but I got the No Python at etc etc. Reinstalled 3.11.4 and got the ModuleNotFoundError: etc. again
Edit: I'm not sure exactly what changed, but I had run the command py -m pip install requests, tried to run the script again but failed, and tried completely removing python from my system and starting over, installing the 3.11.4 version and re-downloading the repository, which seems to have fixed my issue.
